components/style/values/generics/gecko.rs

```diff
@@ -0,0 +1,55 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+//! Generic types for legacy Gecko-only properties that should probably be
+//! unshipped at some point in the future.
+
+use std::fmt;
+use style_traits::ToCss;
+
+/// A generic value for scroll snap points.
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+#[derive(Clone, Copy, Debug, HasViewportPercentage, PartialEq, ToComputedValue)]
+pub enum ScrollSnapPoint<LengthOrPercentage> {
```

Comment:
This is in a gecko-only module, so no need for a `cfg_attr(feature = "servo"...)`. Though any reason why compiling it for both hurts?